scripts/__preload__.zeek

```diff
@@ -4,6 +4,9 @@ module Spicy;
 
 export {
 # doc-options-start
+    # Constant for testing if Spicy is available.
+    const available = T;
+
```

Comment:
It looks like this won't be available in Zeek bare mode. Could we move this to `bare.zeek` (unsure how that interacts with `consts.bif` currently).

Reply:
Good observation—but it's actually working as expected: in bare mode, plugins aren't active by default, and hence Spicy will indeed not be available. Activation needs to be explicit in bare mode: `zeek -b Zeek::Spicy ...`. I just confirmed that `available` then works correctly. I have also added a test now that checks the 3 cases that we have here.